### ISSUE TITLE
Sampling period str or int

### ISSUE THREAD

**Issue by gareth-j** [openghg/openghg#98]
When passing `sampling_period` to `ObsSurface.read_file` we expect `sampling_period: Sampling period in pandas style (e.g. 2H for 2 hour period, 2m for 2 minute period)`.

This is checked in `_crds.py` - https://github.com/openghg/openghg/blob/4f04dbcd29038b79aad39d6dc860e2b2f12a2554/openghg/modules/_crds.py#L157 as we want to create a `Timedelta` object for the input sampling period. 

In `assign_attributes` though we expect an `int` that we assign to `sampling_period_seconds` in the `time_attributes` dictionary - https://github.com/openghg/openghg/blob/4f04dbcd29038b79aad39d6dc860e2b2f12a2554/openghg/processing/_attributes.py#L280. I'm guessing the creation of the `Timedelta` isn't what we want to do?

**Comment by rt17603** [comment]
Yeah this was something I grappled with when I updated the sampling period code. The problem was that the way the read_file interface was written, it expected a string but a float was expected elsewhere. In the end I opted for keeping it the way it was for simplicity after dancing around it for a bit.

I was thinking that the end state may end up being that we drop "sampling_period" as an input to read_file and have to find a way to derive this from the data for each data type.
